### DISCUSSION TITLE
import SASS/SCSS/CSS to the components and pages directly not as module

### DISCUSSION THREAD

**Discussion by Scorpovi4** [vercel/next.js#29958] [Help]
Based on documentation direct styles import to the next.js like this one is possible only in _app.js,:
`import '../styles.css'`

in the rest components it requires to be with postfix `module` to avoid error but it still not working.

As I see `@zeit/next-css` / `@zeit/next-sass` packages gave that but they don't work for me and unsupported now.

Exist way to import like that `import '../styles.module.css'` and let it works in any way (because it's really convenient) as next.config customizing or with help of third party packages?

**Comment by aafnnp** [comment]
so terrible with module

**Comment by utsav0209** [comment]
@Scorpovi4  Did you find any solution to this?
